places-solution-integrations/target/attach-data.md

```diff
@@ -0,0 +1,76 @@
+# Attaching Places Data to Target Events via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
+
+Once Places is sending in events for entries and exits, you can leverage Rules in Launch to attach your Places data to your Adobe Target SDK events. With your desired property selected in Launch, you can create this type of rule by completing four steps:
+
+- [Create a new `Rule`](#create-a-rule)
+- [Select the `Event` you wish to trigger the rule](#select-an-event)
+- [Add any other `Conditions` required to trigger the action (optional)](#add-conditions)
+- [Select the `Action` to Attach Data and define your payload](#define-the-action)
+
+## Create a Rule
+
+In the `Rules` tab, click the `Create New Rule` button.
+
+If you don't have existing rules for this property, the button will be in the middle of the screen:
```

Comment:
Change to:
"Remember the following information:
*If you do not have existing rules for this property, the button will be in the middle of the screen.
* If your property already has rules, the button will be in the top right hand side of the screen."

Please remove the graphics